name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java

```diff
@@ -180,7 +180,7 @@ public ChartInterval getInverval(Client client, CurrencyConverter converter, Sec
                 case H:
                     List<TransactionPair<?>> tx = security.getTransactions(client);
                     if (tx.isEmpty())
-                        return new ChartInterval(now, now);
+                        return getAllPrices(now, security);
```

Comment:
We could just return `null` as the `ChartInterval` (here and below, where the other `ChartInterval(now, now)` is currently); it seems this one-day interval causes the chart to get that huge an interval. Then at https://github.com/buchen/portfolio/blob/master/name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java#L725-L738, we could catch the `null` and do the same as for a `null` range. We could even put the test for `null` into `ChartRange.createFor()` (return `null` when a `null` interval is passed) and wouldn’t have to do checking twice in this file.

Reply:
Implemented with [18138fe](https://github.com/buchen/portfolio/pull/3196/commits/18138fe0a2a34ae35f9ddce26b7be0d3441f96b7)